deployment/cfn/template_utils.py

```diff
@@ -0,0 +1,113 @@
+import ConfigParser
+import boto.ec2
+import majorkirby as mj
+
+VPC_CIDR = '10.0.0.0/16'
+ALLOW_ALL_CIDR = '0.0.0.0/0'
+
+EC2_REGIONS = [
```

Comment:
Might be worth pulling these down dynamically since we already have API credentials.